tsconfig.json

```diff
@@ -1,18 +1,20 @@
 {
   "compilerOptions": {
-    "target": "es2017",
+    "allowJs": true,
+    "baseUrl": "./",
+    "esModuleInterop": true,
     "experimentalDecorators": true,
-    "removeComments": false,
+    "module": "commonjs",
+    "moduleResolution": "node",
     "noImplicitAny": true,
     "noImplicitThis": true,
-    "moduleResolution": "node",
-    "module": "commonjs",
+    "outDir": "build",
+    "removeComments": false,
     "resolveJsonModule": true,
-    "esModuleInterop": true,
-    "allowJs": true,
-    "skipLibCheck": true
+    "rootDir": "./",
+    "skipLibCheck": true,
+    "target": "es2017"
```

Comment:
Do we need to specify a target? Will this affect the build or just VSCode?
If it doesn't affect the build, then esnext seems more appropriate.